parlai/agents/bert_ranker/helpers.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+from parlai.core.torch_ranker_agent import TorchRankerAgent
+import torch
+try:
+    from pytorch_pretrained_bert.modeling import BertLayer, BertConfig
+    from pytorch_pretrained_bert import BertAdam
+    from pytorch_pretrained_bert import BertModel  # NOQA
+except ImportError:
+    raise Exception(("BERT rankers needs pytorch-pretrained-BERT installed. \n "
+                     "pip install pytorch-pretrained-bert"))
+from parlai.core.utils import _ellipse
+
+
+def add_common_args(parser):
+    """Add command line arguments for this agent."""
+    TorchRankerAgent.add_cmdline_args(parser)
+    parser = parser.add_argument_group('Bert Ranker Arguments')
+    parser.add_argument('--history-length', type=int, default=5,
```

Comment:
Is this different from the role `-histsz` in TA is supposed to play?

Reply:
I don't think it's different. Also it was actually not used, removed.